### PR TITLE
Spanish translation done!

### PR DESCRIPTION
**Pull request by Fcosalvado**
I have started translating, I guess that between tomorrow and Sunday I could be almost finished/finished. Let me know what you think. Cheers!

### CODE REVIEW

File: src/culture/es.js [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
-		"detailalertnotvalidprivatekey": "El texto que has introducido no es una clave privada válida",
-		"detailconfirmsha256": "El texto que has introducido no es una clave privada válida\n\n¿Quieres usar ese texto como si fuera una contraseña y generar una clave privada usando un hash SHA256 de tal contraseña?\n\nAviso: Es importante escoger una contraseña fuerte para evitar ataques de fuerza bruta a fin de adivinarla y robar tus bitcoins.",
-		"bip38alertincorrectpassphrase": "Incorrect passphrase for this encrypted private key.", //TODO: please translate
-		"bip38alertpassphraserequired": "Passphrase required for BIP38 key", //TODO: please translate
+		"detailalertnotvalidprivatekey": "El texto introducido no es una clave privada válida",
+		"detailconfirmsha256": "El texto introducido no es una clave privada válida\n\n¿Quiere usar ese texto como si fuera una contraseña y generar una clave privada usando un hash SHA256 de esa contraseña?\n\nAviso: Es importante escoger una contraseña robusta para evitar ataques de fuerza bruta que buscan descifrarla y robar sus bitcoins.",
+		"bip38alertincorrectpassphrase": "Contraseña incrrecta para esta clave privada encriptada.", // DONE
```
Comment (zquestz) [inline src/culture/es.js]: I think we can remove the `// DONE` comments.

File: src/culture/es.js [hunk 2]
```diff
@@ -1,80 +1,80 @@
...
 		"brainwallet": "Cartera mental",
 		"vanitywallet": "Cartera personalizada",
-		"splitwallet": "Split Wallet", //TODO: please translate
+		"splitwallet": "Cartera separada", // DONE
 		"detailwallet": "Detalles de la cartera",
 
 		// footer html
 		"footerlabeldonations": "Donaciones:",
-		"footerlabeltranslatedby": "Traducción: <b>12345</b>Vypv2QSmuRXcciT5oEB27mPbWGeva",
+		"footerlabeltranslatedby": "Traducción: <b>12345</b>Vypv2QSmuRXcciT5oEB27mPbWGeva", // Should I put my name here?
```
Comment (zquestz) [inline src/culture/es.js]: I think you should add your name (not replace), and add a BCC address. Luckily we shouldn't need to label the addresses, as all the existing translation addresses were created pre fork, so they can also accept BCC =)

Comment (theantnest) [reply src/culture/es.js]: Re BCC BTC address - Good point!

Comment (zquestz) [reply src/culture/es.js]: This is a good point, then we should label them? BCC and BTC?

Comment (theantnest) [reply src/culture/es.js]: I think so

**Comment by Fcosalvado** [comment]
I have finished the spanish translation and reviewed it looking for any grammar mistakes. I would like to see it displayed and running, or at least a mockup of it, to double check spelling and sentence coherence. Also, I changed the overall text, since there were phrases that were somewhat "informal".
Let me know what you think.
Could you point me as to where exactly should I put my name and address?
Cheers!

File: src/culture/es.js [hunk 3]
```diff
@@ -1,112 +1,112 @@
...
+		"generatelabelmovemouse": "Mueve un poco el ratón para crear movimientos al azar...",
+		"generatelabelkeypress": "O BIEN escriba algunos caracteres al azar en este recuadro.",
+		"singlewallet": "Cartera individual", // Are you thinking of Spanish clients or Latin American clients? I ask because for spanish wallet=cartera and for LATAM wallet=billetera.
 		"paperwallet": "Cartera en papel",
-		"bulkwallet": "Direcciones en masa",
+		"bulkwallet": "Cartera con múltiples direcciones", // I have a feeling this name speaks more of its concept... I don't know if BULK translates as accurately to Spanish
 		"brainwallet": "Cartera mental",
 		"vanitywallet": "Cartera personalizada",
-		"splitwallet": "Split Wallet", //TODO: please translate
+		"splitwallet": "Cartera separada",
```
Comment (zquestz) [inline src/culture/es.js]: We are missing a few splitwallet keys! These are from the hungarian file. The english versions are in `src/bitaddress-ui.html`. Just look for the CSS ids.

```
		"splitwallet": "Felosztott tárca",
		"splitlabelthreshold": "A visszaállításhoz szükséges részek minimális száma",
		"splitlabelshares": "Részek száma",
		"splitview": "Előállítás",
```

**Comment by zquestz** [comment]
@Fcosalvado you will need to run `grunt` to build the page. Then view cashaddress.org.html in your browser for testing. The logs and other items will be wrong until I merge my other PRs. Also I left a comment about 3 missing splitwallet keys. I noticed them when fixing the "Split Wallet" translations being broken in my other PR.

File: src/culture/es.js (hunk 2 above)
Comment (zquestz) [reply src/culture/es.js]: Feel free to add your address, make sure you prefix them both with BTC/BCC for clarity. Should do this before we merge.

File: src/culture/es.js (hunk 3 above)
Comment (Fcosalvado) [reply src/culture/es.js]: Here you go, from what I can make from the hungarian version and from src/bitaddress-ui.html:

"splitwallet": "Cartera de múltiples direcciones",
"splitlabelthreshold": "Umbral de divisiones mínimas necesarias para combinar",
"splitlabelshares": "Número de divisiones",
"splitview": "Vista de divisiones",

Let me know if there is anything missing!

Comment (zquestz) [reply src/culture/es.js]: Excellent! I trust your spanish. =)

Comment (zquestz) [reply src/culture/es.js]: Lets go ahead and add those to the PR. =)

**Comment by Fcosalvado** [comment]
I added my info in line 38. Could you check it and let me know if it's correct?
Cheers!

File: src/culture/es.js [hunk 4]
```diff
@@ -1,112 +1,113 @@
...
 		"detailwallet": "Detalles de la cartera",
 
 		// footer html
 		"footerlabeldonations": "Donaciones:",
 		"footerlabeltranslatedby": "Traducción: <b>12345</b>Vypv2QSmuRXcciT5oEB27mPbWGeva",
+		"footerlabeltranslatedby": "Traducción: <b>Francisco Salvado</b>34asTQaYjY1wViisS7ydueY3Kx3Cw5dVLW",
```
Comment (zquestz) [inline src/culture/es.js]: That doesn't look like a valid BCC address. Should start with 1.

Comment (Fcosalvado) [reply src/culture/es.js]: I'm using a BCC wallet from bcc-wallet.btc.com, I just checked and all my addresses start with 3 :o

Comment (zquestz) [reply src/culture/es.js]: Interesting... I haven't used that wallet. I thought they had to start with 1, guess I was wrong. =\

Comment (zquestz) [reply src/culture/es.js]: I thought segwit addresses started with 3... I am going to look around. Something is strange.

Comment (Fcosalvado) [reply src/culture/es.js]: I have tested it sending and receiving BCC and so far no problems. Is there another wallet that you recommend for the time being?

Comment (zquestz) [reply src/culture/es.js]: NM... multi sig addresses in BCC start with 3. We are good. Whew. =)

Comment (Fcosalvado) [reply src/culture/es.js]: Great! Let me know if you need anything else, and when the wallet will be up and running.
Cheers!

Comment (zquestz) [reply src/culture/es.js]: Last tiny NIT, maybe a space between your name and the address?

**Comment by Fcosalvado** [comment]
Done! I added a space between my name and the address. Also spotted some tiny misspelling and corrected it. Cheers!

**Pull request merged**